test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1124,21 +1112,27 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
     #region Client Certificate Authentication
 
     It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"  {
-        $uri = 'https://localhost:8083/'
-        $result = Invoke-WebRequest -Uri $uri -SkipCertificateCheck
-        $failedMessage = 'Status: Failed'
-        $result.RawContent  | Should Match ([regex]::Escape($failedMessage ))
+        $uri = '{0}/Cert/' -f (Get-WebListenerUrl -Https)
+        $result = Invoke-WebRequest -Uri $uri -SkipCertificateCheck | 
+            Select-Object -ExcludeProperty Content |
+            ConvertFrom-Json
+        
+        $result.Status  | Should Be 'FAILED'
+        $result.Status  | Should Not Be 'OK'
     }
     
     # Test skipped on macOS pending support for Client Certificate Authentication
     # https://github.com/PowerShell/PowerShell/issues/4650
     It "Verifies Invoke-WebRequest Certificate Authentication Successful with -Certificate" -skip:$IsOSX {
-        $uri = 'https://localhost:8083/'
-        $certificate = GetSelfSignedCert
-        $result = Invoke-WebRequest -Uri $uri -Certificate $certificate -SkipCertificateCheck
-        $successMessage = 'Status: OK'
-        $result.RawContent | Should Match ([regex]::Escape($successMessage))
-        $result.RawContent | Should Match $certificate.Thumbprint
+        $uri = '{0}/Cert/' -f (Get-WebListenerUrl -Https)
+        $certificate = Get-TestClientCertificate
```

Comment:
I think we should have the same prefix for all cmdlets from `WebListener` module.